### ISSUE TITLE
[BUG] no errors reported when running on current directory (`.`)

### ISSUE THREAD

**Issue by DetachHead** [dart-code-checker/dart-code-metrics#799]
* **Dart code metrics version:** 4.13.0
* **Dart sdk version:** 2.16.2

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
analyzer:
  plugins:
    - dart_code_metrics
  language:
    strict-casts: true
    strict-inference: true
    strict-raw-types: true
  exclude:
    - lib/generated_plugin_registrant.dart
```

</details>

**What did you do? Please include the source code example causing the issue.**

```
>flutter pub run dart_code_metrics:metrics analyze .
✔ no issues found!

>flutter pub run dart_code_metrics:metrics analyze lib
lib\screens\report.dart:
STYLE   Block is empty. Empty blocks are often indicators of missing code.
        lib\screens\report.dart:85:29
        no-empty-block : https://dartcodemetrics.dev/docs/rules/common/no-empty-block

STYLE   Block is empty. Empty blocks are often indicators of missing code.
        lib\screens\report.dart:90:27
        no-empty-block : https://dartcodemetrics.dev/docs/rules/common/no-empty-block

STYLE   Block is empty. Empty blocks are often indicators of missing code.
        lib\screens\report.dart:104:29
        no-empty-block : https://dartcodemetrics.dev/docs/rules/common/no-empty-block
```

**What did you expect to happen?**
same errors when running in `.`

**What actually happened?**
no errors found

**Comment by incendial** [comment]
@DetachHead have you tried calling with `./`? What's the output?

**Comment by DetachHead** [comment]
```
>flutter pub run dart_code_metrics:metrics analyze ./
✔ no issues found!
```

**Comment by incendial** [comment]
Available in 4.16.0! 🚀

**Comment by DetachHead** [comment]
@incendial it doesn't seem to be fixed for me

```
>flutter pub run dart_code_metrics:metrics analyze .
✔ no issues found!

>flutter pub run dart_code_metrics:metrics analyze ./
✔ no issues found!

>flutter pub run dart_code_metrics:metrics analyze test
test\test.dart:
STYLE   Block is empty. Empty blocks are often indicators of missing code.
        test\test.dart:11:12
        no-empty-block : https://dartcodemetrics.dev/docs/rules/common/no-empty-block

>flutter pub run dart_code_metrics:metrics --version   
Dart Code Metrics version: 4.16.0
```

**Comment by incendial** [comment]
@DetachHead That's strange, coz I tested it locally and everything was fine. I'll retest it come back

**Comment by DetachHead** [comment]
@incendial any update? let me know if you need any more information from me to reproduce

**Comment by incendial** [comment]
@DetachHead sorry, not yet. Plan to check it today

**Comment by incendial** [comment]
@DetachHead tested it locally and everything works as expected. Could you create a reproduction example?
